pkg/gofr/gofr.go

```diff
@@ -39,6 +39,7 @@ const (
 	shutDownTimeout        = 30 * time.Second
 	gofrTraceExporter      = "gofr"
 	gofrTracerURL          = "https://tracer.gofr.dev"
+	defaultTraceRatio      = 1
```

Comment:
I don't see this constant being used anywhere.

Reply:
Ahh yes. I will remove it.